docs/content/dependencies/metadata-storage.md

```diff
@@ -32,7 +32,9 @@ Derby is the default metadata store for Druid, however, it is not suitable for p
 [MySQL](../development/extensions-core/mysql.html) and [PostgreSQL](../development/extensions-core/postgresql.html) are more production suitable metadata stores.
 
 <div class="note caution">
-Derby is not suitable for production use as a metadata store. Use MySQL or PostgreSQL instead.
+The Metadata Storage stores the entire metadata which is essential for a Druid cluster to work.
+For production clusters, consider using MySQL or PostgreSQL instead of Derby.
+Also, it's highly recommended to set up a high availability environment.
```

Comment:
It might be worth stressing the importance of not losing this data, since there is no longer a way to rebuild it in the event of a failure.

Reply:
Thanks. I tried..